packages/commonwealth/client/scripts/controllers/app/webWallets/walletconnect_web_wallet.ts

```diff
@@ -47,35 +52,15 @@ class WalletConnectWebWalletController implements IWebWallet<string> {
     }
   }
 
-  public async signMessage(message: string): Promise<string> {
-    const signature = await this._web3.eth.sign(message, this.accounts[0]);
-    return signature;
-  }
-
-  public async signLoginToken(validationBlockInfo: string): Promise<string> {
-    const sessionPublicAddress = app.sessions.getOrCreateAddress(this._chainInfo.node.ethChainId);
-    const msgParams = await constructTypedMessage(
-      this.accounts[0],
-      this._chainInfo.node.ethChainId || 1,
-      sessionPublicAddress,
-      validationBlockInfo,
-    );
+  public async signCanvasMessage(account: Account, canvasMessage: CanvasData): Promise<string> {
```

Comment:
TODO: us to test WC issue.

Reply:
Got WalletConnect working locally, it was probably a misconfigured API key.